src/main/java/Main.java

```diff
@@ -0,0 +1,21 @@
+import org.json.JSONObject;
+import org.json.simple.parser.ParseException;
+
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) {
+        String inputFilename = "data_example.json.gz";
+        String outPutFileName = "data_example.xml";
+        try {
+            JSONObject jsonObject = ReadGZIP.readGZIPToJsonObject(inputFilename);
+            byte[] xmlData = ConvertJsonToXml.convertJsonToXmlRawBytes(jsonObject);
+            OutputStreamWriteXml outputStreamWriteXml = new OutputStreamWriteXml(outPutFileName);
+            outputStreamWriteXml.writeXml(xmlData);
```

Comment:
You missed the point of outputstream..
you already have the xml as input, use this to output as stream instead of giving the whole byte array as parameter for outputstream.write function